src/apprt/gtk/style.css

```diff
@@ -41,6 +41,22 @@ window.ssd.no-border-radius {
   border-radius: 0 0;
 }
 
+paned > separator {
```

Comment:
This should be combined with the `paned > separator` CSS section later in the file.

Reply:
Probably need to run this through `prettier` as well.

Reply:
Done.